test/Microsoft.AspNet.Mvc.FunctionalTests/ViewEngineTests.cs

```diff
@@ -290,7 +290,7 @@ public static IEnumerable<object[]> RazorViewEngine_UsesExpandersForLayoutsData
         // This change ensures that _ViewStart files correctly inherits directives from parent _ViewStarts
         // which guarantees that paths flow correctly through MvcRazorHost.
         [Fact]
-        public async Task ViewStartsCanUseDirectivesInjectedFromParentViewStarts()
+        public async Task ViewStartsCanUseDirectivesInjectedFromParentGlobals()
```

Comment:
Supposed to be `ViewStarts`? Was looking at the files below and didn't look like it was `ViewStart` based. That being said though it is important to validate the scenario of utilizing things like `@inject` properties in a `ViewStart`